src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2343,16 +2343,17 @@ namespace HVACUnitarySystem {
 			if ( MSHPIndex > 0 ) {
 				for ( Iter = DesignSpecMSHP( MSHPIndex ).NumOfSpeedHeating; Iter >= 1; --Iter ) { // use reverse order since we divide by HeatVolumeFlowRate(max)
 					if ( DesignSpecMSHP( MSHPIndex ).HeatingVolFlowRatio( Iter ) == AutoSize ) {
-						if ( UnitarySystem( UnitarySysNum ).HeatingCoilType_Num == Coil_HeatingElectric_MultiStage || UnitarySystem( UnitarySysNum ).HeatingCoilType_Num == Coil_HeatingGas_MultiStage ) {
+						if ( UnitarySystem( UnitarySysNum ).ControlType == SetPointBased && ( UnitarySystem( UnitarySysNum ).HeatingCoilType_Num == Coil_HeatingElectric_MultiStage || UnitarySystem( UnitarySysNum ).HeatingCoilType_Num == Coil_HeatingGas_MultiStage ) ) {
 							DesignSpecMSHP( MSHPIndex ).HeatingVolFlowRatio( Iter ) = 1.0;
```

Comment:
Only force full flow for set point control with MS gas/elec coils. Load based control should be OK to allow changes in flow rate. Variable flow, variable capacity does not work for SP control since outlet air temperature does not change with PLR. For Q=MCpDeltaT, if Q AND Mdot change proportionally, DetlaT does not change, hence no way to control SP based systems. Flow needs to be constant for SP based control so that outlet air temperature changes with PLR.